### DISCUSSION TITLE
Symfony 2.8 password hashing

### DISCUSSION THREAD

**Discussion by dhillon-lovey3110** [symfony/symfony#42802] [Q&A]
I want to use symfony 2.8 db in some project. But i am now sure how I can match the old passwords in php core. Are those classes available to use separately or any simple php code to use. Sorry I am new to symfony. Looking for some help thanks in advance. Sorry for any typing mistakes.

**Comment by ghost** [comment]
I don't understand the language you are talking.

**Comment by javiereguiluz** [comment]
My guess is that you want some PHP application to use the passwords encoded/hashed by a Symfony 2.8 password. That's has already been asked and answered in another discussion (#42493).

To do so, you need to mimic this method, which is the one used by Symfony to hash things: https://github.com/symfony/symfony/discussions/42493#discussioncomment-1162910
